src/bepasty/apis/rest.py

```diff
@@ -214,7 +214,7 @@ def err_incomplete(self, item, error):
 
     def response(self, item, name):
         return jsonify({'uri': url_for('bepasty_apis.items_detail', name=name),
-                        'file-meta': dict(item.meta)})
+                        'file-meta': filter_internal(dict(item.meta))})
```

Comment:
guess you don't need the `dict()` here, `filter_meta` creates a new dict.

Reply:
looks like work without dict()